src/services/apiService.ts

```diff
@@ -46,7 +47,8 @@ class ApiService {
 
     public removeToken = (): void => localStorage.removeItem("token");
 
-    public loginWithCredentials = (data: ILoginRequestPayload) => {
+    // changed from ILoginRequestPayload to ISignIn
```

Comment:
If `ILoginRequestPayload` is no longer used, it can be removed

Reply:
Actually I was talking with Johanna on slack and anyways we probably will need two interfaces, one for the form `ISignIn` with the checkbox (rememberMe) and second one for the `loginWithCredentials` function.